### PR TITLE
Knshq tailwind migration

### PR DESCRIPTION
**Pull request by KNSHQ**

### CODE REVIEW

File: front_end_project/package.json
Comment (mukundnivas) [inline front_end_project/package.json:1]: Could you justify on your requirement for an external 3rd party library?

File: front_end_project/src/pages/about/aboutUs.jsx
Comment (mukundnivas) [inline front_end_project/src/pages/about/aboutUs.jsx:1]: Is it an absolute necessity to use a 3rd party lib here? Its extremely hard to maintain when 3rd party libraries are used. If you still believe so, please advise.

**Review by mukundnivas — CHANGES_REQUESTED** [review]
Would 3rd party packages be necessary here? When using external library we’re putting ourselves at risk - it’s impossible to know from the documentation or readme of a project if it’s going to be easy to set up and if it’s going to work how you need it too.

If you still believe it would be an necessity, please advise so and we would require a thorough doco on implementation of such components.

**Review by Aaron-Vandersluys-Code — COMMENTED** [review]
I would also like to point out that the about us page has also been redesigned and reimplemented in another task, thus no changes are necessary at the time being. As @mukundnivas has mentioned, some of these changes likely require further investigation beyond the scope of the time we have left to make changes. 

I think for now this pull request will be closed, but as you are a junior you will have the opportunity to revisit this task in the next trimester. Ensure you keep these code changes for your evidence portfolio in Capstone A.

**Pull request closed without merging**